docs/07-my-first-awesome.md

```diff
@@ -25,7 +25,7 @@ have not opened any programs. On the top right you see the time/date and a
 symbol showing the current layout. You can also click on the symbol to change
 the active layout.
 
-One of awesome's big advantages over other tiling window managers is its good
+One of awesome big advantages over other tiling window managers is its good
```

Comment:
Why was the old version of this wrong? It is awesome which has the big advantages, so `'s`, no?